### PR TITLE
ELKS BIOS reduction and add color to text

### PR DESCRIPTION
**Pull request by ghaerr**
Removes functions never called by ELKS (Direct, BIOS or Headless Console) from ROM BIOS.

Tested and works on Direct and BIOS Console.

BTW, ELKS BIOS Console runs well on stdio backend (both ELKS and PCXTAT targets).

This will allow @cocus and others to understand the minimal BIOS required for porting ELKS. Will be updating ELKS porting-guide.txt on ELKS as well.

### CODE REVIEW

File: rom-elks.c [hunk 1]
```diff
@@ -393,25 +348,6 @@ static int int_13h ()
...
-
-		default:
-			flag_set (FLAG_CF, 1);
-
-		}
-
-	return 0;
-	}
-
-
```
Comment (mfld-fr) [inline rom-elks.c:414]: https://github.com/jbruchon/elks/commit/4ef5e84e2d8ea3fcdd213da9e0c5a92c30712be5 ?

Comment (ghaerr) [reply rom-elks.c:414]: Thanks for pointing that out. We could leave it in, but as you can see in the original commit, that code was never updated from BCC to gcc compiler, so INT 15h will never be called. IMO, ELKS APM is completely inoperational at this time. I figure we should remove it until it is actually compilable by ELKS. (It is saved in rom-pcxtat.c of course).

Comment (mfld-fr) [reply rom-elks.c:414]: Okay...

**Comment by ghaerr** [comment]
@mfld-fr : Check this out!

QEMU
QEMU

EMU86
EMU86

**Comment by mfld-fr** [comment]
It is ever better with colors :smile:

**Comment by ghaerr** [comment]
> Don't forget to squash your commits, 'Cleanup' / "More cleanup" are not really meaningful titles for later digging if needed...

Thanks, I will in the future. I got a little excited and added the EGA in 20 minutes, then needed some cleanup :)

**Comment by mfld-fr** [comment]
Not in the future, but starting from this PR, please, because the commit log starts to have too much unaccurate titles, and I would like to keep the house clean, and the history understandable at a glance.

**Comment by ghaerr** [comment]
Ok, I have squashed the cleanup commits, thanks.

**Pull request merged**